stubs/sublime.pyi

```diff
@@ -2614,7 +2614,7 @@ class Phantom:
     ) -> None:
         ...
 
-    def __eq__(self, rhs: 'Phantom') -> bool:
+    def __eq__(self, rhs: 'Phantom') -> bool:  # type: ignore[override]
```

Comment:
```suggestion
    def __eq__(self, rhs: Phantom) -> bool:  # type: ignore[override]
```

I missed those quoted annotations earlier. It's not necessary to use quotes for the types in stub files, even for forward references.